scripts/publishing.gradle

```diff
@@ -1,60 +0,0 @@
-apply plugin: 'com.bmuschko.nexus'
```

Comment:
This script is just removed?

Reply:
Yeah that was the old maven release plugin, but now we go through the new flow.